recipes/stages/_base_/data/pipelines/coco_ote_yolox_pipeline.py

```diff
@@ -0,0 +1,40 @@
+dataset_type = 'CocoDataset'
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+img_scale = (640, 640)
+
+train_pipeline = [
+    dict(type='Mosaic', img_scale=img_scale, pad_val=114.0),
+    dict(
+        type='RandomAffine',
+        scaling_ratio_range=(0.5, 1.5),
+        border=(-img_scale[0] // 2, -img_scale[1] // 2)),
+    dict(
+        type='PhotoMetricDistortion',
+        brightness_delta=32,
+        contrast_range=(0.5, 1.5),
+        saturation_range=(0.5, 1.5),
+        hue_delta=18),
+    dict(type='RandomFlip', flip_ratio=0.5),
+    dict(type='Resize', keep_ratio=True),
+    dict(type='Pad', pad_to_square=True, pad_val=114.0),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_bboxes', 'gt_labels'])
+]
+
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='MultiScaleFlipAug',
+        img_scale=(416, 416),
+        flip=False,
+        transforms=[
+            dict(type='Resize', keep_ratio=True),
+            dict(type='RandomFlip'),
+            dict(type='Pad', size=(416, 416), pad_val=114.0),
```

Comment:
Why use 114.0 as pad_val? Is it common sense for detection tasks? It's just a question.

Reply:
I just followed mmdetection implementation. But It should be consider defining proper pad size for Mosaic augmentation. As far as I know, mosaic augmentation is very common to YOLO style models. I'll check pad value
from original yolo implementation